README.md

```diff
@@ -1,3 +1,47 @@
 # receipe_app
 
-A new Flutter project.
+How to contribute to this open source project
+ Prerequisites
+* You will need to have flutter set up on your system  [https://docs.flutter.dev/get-started/install]
+* You Should know this project was build using stacked as a statemanagement tool  check this for info on stacked[https://stacked.filledstacks.com/docs/getting-started/overview/]
```

Comment:
`build` to `built`, `statemanagement` to `state management`

Reply:
ok, effected.
you can check.\

Reply:
missed the first one

Reply:
okay